dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -170,14 +170,14 @@
 		<AD id="timestampFormat" name="%timestampFormat" description="%timestampFormat.desc" required="false" type="String" />
 		<!-- End VMM ported Entries -->	
 
-	<AD id="bindAuthMechanism" name="internal" description="internal use only" required="false" type="String">
+	<AD id="bindAuthMechanism" name="%bindAuthMechanism" description="%bindAuthMechanism.desc" required="false" type="String" ibm:beta="true">
```

Comment:
The previous code, do we have default for simple?

Reply:
We did not have a default at all. I was planning to set the default of bindAuthMechanism to simple when we GA it.